snap/snapenv/snapenv_test.go

```diff
@@ -94,6 +94,7 @@ func (ts *HTestSuite) TestBasic(c *C) {
 		"SNAP_ARCH":          arch.DpkgArchitecture(),
 		"SNAP_LIBRARY_PATH":  "/var/lib/snapd/lib/gl:/var/lib/snapd/lib/gl32:/var/lib/snapd/void",
 		"SNAP_REEXEC":        "",
+		"SNAP_UID":           fmt.Sprint(sys.Geteuid()),
```

Comment:
we have some spread tests (specifically `tests/main/snap-env/task.yaml`) that will need corrections

Reply:
I'll check it right now...

Reply:
I think this should work...